Shopstargram/css/_christmas.scss

```diff
@@ -0,0 +1 @@
+$color-white: red;
```

Comment:
nie jest to jakas ambitna zmiana styli :P, ale powiedzmy ze podpielas poprawnie theme